mix.lock

```diff
@@ -11,14 +11,13 @@
   "dialyxir": {:hex, :dialyxir, "0.5.1", "b331b091720fd93e878137add264bac4f644e1ddae07a70bf7062c7862c4b952", [:mix], [], "hexpm"},
   "distillery": {:hex, :distillery, "1.5.3", "b2f4fc34ec71ab4f1202a796f9290e068883b042319aa8c9aa45377ecac8597a", [:mix], [], "hexpm"},
   "edown": {:hex, :edown, "0.8.1", "7333b6f6b7bbc736c263e9ceb8261667d7c8f7d92e251829f1a9fe8f24e7b694", [:rebar3], [], "hexpm"},
-  "erlexec": {:hex, :erlexec, "1.7.1", "6ddbd40fa202084ed0bdaf95a50c334acaa5644ae213b903cd4094a78ae79734", [:rebar3], [], "hexpm"},
-  "ex_parameterized": {:hex, :ex_parameterized, "1.3.1", "c122a967b1265e842b61bdc57dcff65918802c8853ceacc0a7f6bf604e6561cd", [:mix], [], "hexpm"},
   "eleveldb": {:hex, :eleveldb, "2.2.20", "1fff63a5055bbf4bf821f797ef76065882b193f5e8095f95fcd9287187773b58", [:rebar3], [], "hexpm"},
   "enacl": {:git, "https://github.com/aeternity/enacl.git", "2f50ba6289f2f2d9fef05d22a396c0bab4f64149", [ref: "2f50ba6"]},
   "enoise": {:git, "https://github.com/aeternity/enoise.git", "6d793b711854a02d56c68d9959e1525389464c87", [ref: "6d793b711854a02d56c68d9959e1525389464c87"]},
   "erl_base58": {:hex, :erl_base58, "0.0.1", "37710854461d71df338e73c65776302db41c4bab4674d2ec134ed7bcfc7b5552", [:rebar3], [], "hexpm"},
   "erlexec": {:hex, :erlexec, "1.7.5", "3c09adf19b217c24c31040c489a3f8b5bc28ba907749c211f2ab81b431e1b990", [:rebar3], [], "hexpm"},
   "ex_json_schema": {:hex, :ex_json_schema, "0.5.7", "14a1bcd716e432badb19e5f54fd0f3f0be47b34d1b5957944702be90d66a6cf6", [:mix], [], "hexpm"},
+  "ex_parameterized": {:hex, :ex_parameterized, "1.3.1", "c122a967b1265e842b61bdc57dcff65918802c8853ceacc0a7f6bf604e6561cd", [:mix], [], "hexpm"},
```

Comment:
I think changes to mix.lock shouldn't be included.

Reply:
This way we are all at the same version.

Reply:
ok, We can probably leave it as is. After careful inspection I see ex_parametrized is just moved and erlexec removed.